README.md

```diff
@@ -3,12 +3,11 @@ Server Log v1.2.0
 A thread safe logging system for servers. Serverlog can log to files and to the console(with pretty colours) depending on what you specify. Logfiles will be separated by dates and can also manage how many logfiles you want on the server at any one time.
 
 ## Import
-Go get the package like this
-```
-go get github.com/canopener/serverlog
-```
-Import like this
-```
+Go get the package like this  
+> `go get github.com/canopener/serverlog`
```

Comment:
Perhaps just `$ go get github.com/canopener/serverlog` would be better. The "$" at the start makes it clear it's run in a shell.